ansible/roles/ocp-workload-integreatly-rhte-2019-lab/tasks/create-tenants.yml

```diff
@@ -22,37 +22,40 @@
 - debug:
     msg: 3scale master access token is {{ threescale_master_access_token }}
 
-- name: Wait until 3scale API is available 
-  uri: 
+- name: Wait until 3scale API is available
+  uri:
     url: https://{{ threescale_host }}
     method: HEAD
-    validate_certs: no
+    validate_certs: false
   register: wait_threescale_result
   until: wait_threescale_result is succeeded
-  ignore_errors: yes
+  ignore_errors: true
   retries: 10
   delay: 60
 
-- name: Wait until 3scale API is available 
-  uri: 
+- name: Wait until 3scale API is available
+  uri:
     url: https://{{ threescale_host }}
     method: HEAD
-    validate_certs: no
+    validate_certs: false
   register: wait_threescale_result
   until: wait_threescale_result is succeeded
-  ignore_errors: yes
+  ignore_errors: true
   retries: 10
   delay: 60
 
 - name: Create Tenant {{ item }}
   uri:
     url: https://{{ threescale_host }}/master/api/providers.xml
     method: POST
-    return_content: yes
+    return_content: true
     headers:
       Content-Type: "application/x-www-form-urlencoded"
-    body: "access_token={{threescale_master_access_token}}&org_name={{ userno }}&username={{ userno }}&email=admin%40{{ userno }}.com&password=Password1"
-    validate_certs: no
+    body: >-
```

Comment:
This can be made more readable or you can just disable yamllint for this. Either one works.

```
body: >-
  {{ 'access_token={0}&org_name={1}&username={1}&email=admin%40{1}.com&password=Password1'.format(
  threescale_master_access_token, userno) }}
```